Apps/WebClient/src/ClientApp/app/components/timeline/commentSection.vue

```diff
@@ -174,5 +202,53 @@ export default class CommentSectionComponent extends Vue {
         this.isLoadingComments = false;
       });
   }
+
+  private updateComment(): void {
```

Comment:
should the update comment live in the comment instead of the list container?

Reply:
Same conversation to be had about deleteComment - made a comment about it below